services/requester/cadence/run.cdc

```diff
@@ -16,7 +16,7 @@ transaction(hexEncodedTx: String) {
         )
         assert(
             txResult.status == EVM.Status.failed || txResult.status == EVM.Status.successful,
-            message: "failed to execute evm transaction: ".concat(txResult.errorCode.toString())
+            message: "evm_error=".concat(txResult.errorCode.toString()).concat("\n")
```

Comment:
We might want to consider using the human-friendly `errorMessage` field instead. See: https://github.com/onflow/flow-go/blob/master/fvm/evm/stdlib/contract.cdc#L270-L272

Reply:
yes I agree, just this wasn't available when I started. Will update before we merge